packages/rocketchat-integrations/server/lib/triggerHandler.js

```diff
@@ -84,7 +84,7 @@ RocketChat.integrations.triggerHandler = new class RocketChatIntegrationHandler
 		}
 
 		if (data) {
-			history.data = data;
+			history.data = { ...data };
```

Comment:
Sadly, this does not work as you expect it to work. So, we need to revert this change:
![image](https://user-images.githubusercontent.com/850391/39268310-1837b6d2-4895-11e8-9ba2-91642754ba20.png)

Reply:
@graywolf336 the _clone doesn't do the same? Copying only the first level of the object?